### DISCUSSION TITLE
Support for onclick()?

### DISCUSSION THREAD

**Discussion by jcoonrod** [servo/servo#32892] [Q&A] [answered]
I notice that servo doesn't support onclick, at least not as defined in the w3schools demo - https://www.w3schools.com/jsref/event_onclick.asp - I've written some simple webgames that depend on onclick - is there a supported workaround in servo?

**Comment by jdm** [comment, marked as answer]
Servo does dispatch click events. If there are cases that do not work as you expect, please file issues with minimal test cases attached!

**Reply by jdm** [reply]
Additionally, I tried several examples from the w3schools page and they worked correctly in Servo.

**Comment by jcoonrod** [comment]
Attached is the code from
https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onclick that
doesn't work in the current servo.

**Reply by jdm** [reply]
It does actually work, but there's a delay while a bunch of XMLHttpRequest requests are processed, and then you need to scroll down to observe the "Hello world".
